src/openloco/graphics/image_ids.h

```diff
@@ -53,7 +53,8 @@ namespace openloco::image_ids
     constexpr uint32_t rotate_object = 2365;
 
     constexpr uint32_t stop_start = 2369;
-
```

Comment:
This whitespace probably shouldn't be removed, as the constants aren't consecutive (yet).